utils/preferences/src/extensions/unicode/keywords/currency.rs

```diff
@@ -0,0 +1,28 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::extensions::unicode::errors::PreferencesParseError;
+use crate::struct_keyword;
+use icu_locale_core::{extensions::unicode::Value, subtags::Subtag};
+use tinystr::TinyAsciiStr;
+
+struct_keyword!(
+    Currency,
```

Comment:
```suggestion
    CurrencyType,
```

I'll consider this one optional if you think this struct should become our standard type for representing currencies in ICU4X.

Reply:
No, I'd argue that once we have a standard type for representing currencies this struct should be moved from `struct CurrencyType(TinyAsciiStr<3>)` to be `struct CurrencyType(Currency)` - and it'll be transparent.